### PR TITLE
Update the README.md with recent changes

### PR DESCRIPTION
**Pull request by jeremycline**
Signed-off-by: Jeremy Cline <jeremy@jcline.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,74 @@
...
 
 ## Install
 
+To get ``fmn.sse`` directly from PyPi you can use pip:
+
+```
+pip install fmn.sse
+```
+
+If you're using Fedora, you can install it with DNF:
```
Comment (pypingou) [inline README.md:19]: is it in epel?

**Comment by pypingou** [comment]
👍 for me

File: README.md (hunk 1 above)
Comment (jeremycline) [reply README.md:19]: I added instructions for EPEL 7

**Comment by pypingou** [comment]
Still good for me 👍

**Pull request merged**